rust/src/domains/ffi.rs

```diff
@@ -175,10 +175,15 @@ pub extern "C" fn opendp_domains__atom_domain(
         Some(())
     }
 
-    if let Some(_) = dispatch!(in_set, [(T, @floats)]) {
-        dispatch!(monomorphize_float, [(T, @floats)], (bounds, nullable))
-    } else if let Some(_) = dispatch!(in_set, [(T, @integers)]) {
-        dispatch!(monomorphize_integer, [(T, @integers)], (bounds, nullable))
+    if let Some(_) = dispatch!(in_set, [(T, [f32, f64])]) {
+        dispatch!(monomorphize_float, [(T, [f32, f64])], (bounds, nullable))
+    } else if let Some(_) = dispatch!(in_set, [(T, [u8, u16, u32, u64, i8, i16, i32, i64, usize])])
```

Comment:
Add a comment reminding us why `isize` isn't in this list?

Reply:
I guess it wouldn't hurt. It's pretty niche.